apps/admin/frontend/src/screens/write_ins_adjudication_screen.tsx

```diff
@@ -692,9 +710,9 @@ export function WriteInsAdjudicationScreen({
                 variant={currentWriteInMarkedInvalid ? 'secondary' : 'regular'}
                 disabled={!isWriteInDetailViewFresh}
               >
-                Mark Write-In Invalid
+                <Icons.DangerX /> Mark write-in invalid
               </Button>
-            </Prose>
+            </div>
```

Comment:
I wonder if it might be confusing for these buttons to start offscreen when the candidate list is long? Maybe they should be in a fixed footer instead of inside the scrollable content

That would probably work best if they were side-by-side - maybe changing the text to "Add candidate" and "Mark invalid" would do it

Reply:
Had a similar thought and initially started heading down the path of having them at the top of the section, but I'd already overshot my time box for this task.

I like the idea idea of a sticky footer even more - will check in with @adghayes when he's back on how common a case it is to have a long list of write-ins to see if we should prioritise a task to address